Cargo.toml

```diff
@@ -9,4 +12,4 @@ repository = "https://github.com/Barugon/egui_file"
 version = "0.4.1"
 
 [dependencies.egui]
-version = "0.19"
+version = "0.20.1"
```

Comment:
Please make this "0.20" so that minor bug fix updates are automatically included.

Reply:
done-didn't think about that :/